operators/endpointmetrics/controllers/observabilityendpoint/metrics_collector.go

```diff
@@ -427,7 +463,7 @@ func getMetricsAllowlist(ctx context.Context, c client.Client,
 	}
 
 	cmList := &corev1.ConfigMapList{}
-	err = c.List(ctx, cmList, &client.ListOptions{})
+	_ = c.List(ctx, cmList, &client.ListOptions{})
```

Comment:
Shouldn't we check for errors here?

Reply:
Agreed. We were ignoring the error before too. There are other places in this method where error is only logged, but not returned. I will create a separate issue to address this.